README.md

```diff
@@ -299,6 +299,27 @@ In case your browser stoped working delete/comment the following line in the scr
  `os.system("TASKKILL /F /IM " + brow + ".exe")`
 
 
+* ### Twitter Sentiment Analysis
+
+A python script that goes through the twitter feeds and calculates the sentiment of the users based on Twitter feeds on the topic of Demonetization in India. 
+Sentiments are calculated to be positive, negative or neutral. Various graphs give a pictorial analysis of the hashtags used. 
+
+ 
+#### Installation
+
+pip install random
+pip install re
+pip install textblob
+pip install collections
+pip install matplotlib
+pip install numpy
+

```

Comment:
Please remove these lines and make a file named analyseTweets-requiremets,txt add modules in that file and add a command to run that file. see [here](https://github.com/Logan1x/Python-Scripts#facebook-auto-post) and [here](https://github.com/Logan1x/Python-Scripts/blob/master/bin/facebook-auto-post.requirements.txt).